nise/generators/ocp/ocp_generator.py

```diff
@@ -88,20 +89,54 @@
     "namespace",
     "namespace_labels",
 )
+OCP_ROS_USAGE_COLUMN = (
```

Comment:
We should also include the report period here for completeness:
```
    "report_period_start",
    "report_period_end",
```

Reply:
Done